.github/workflows/container-autobuild.yml

```diff
@@ -11,6 +11,7 @@ on:
       - '.bazelversion'
       - 'rust-toolchain.toml'
       - 'gitlab-ci/container/**'
+  workflow_dispatch:
```

Comment:
I think this won't work. We need PR to have references updated.

Reply:
Yeah I was thinking we could make a PR with an empty commit, then manually trigger CI from that branch and it would update the refs.